index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>porfolio project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+ <header class="header-section">
+<a class="logo">My Logo</a>
```

Comment:
You removed the underlining but the text is black now. Please make sure the color and font matches with Figma template.
Figma design
<img width="529" alt="Screenshot 2023-01-19 at 8 06 25 PM" src="https://user-images.githubusercontent.com/25887253/213512584-e58cd721-ca9e-42f4-a1e1-68850b10609a.png">
Your design:
![image](https://user-images.githubusercontent.com/25887253/213512901-2ace5f4f-be8e-4632-b401-8c45cfd5f298.png)